generator/generator_test.cc

```diff
@@ -144,3 +144,8 @@ TEST_F(GeneratorTest, GenerateServicesSuccess) {
 }  // namespace generator
 }  // namespace cloud
 }  // namespace google
+
+int main(int argc, char** argv) {
```

Comment:
If we add a `main()`, shouldn't we also be changing a `gtest_main` to a `gtest`?